webpack.config.js

```diff
@@ -21,21 +27,59 @@ module.exports = (env) => {
       /\bwebpack\/buildin\b/
     ];
 
-    filename += '.bundle';
+    babelOptions = {
+      sourceType: 'unambiguous',
+      ignore: [/\/core-js/],
+      presets: [
+        ['@babel/preset-env',
+          {
+            debug: false, /* set to `true` if you start banging your head against the wall */
+            targets: [
+              'last 2 versions',
+              '> 5%',
+              'IE 11',
+              'not dead'
+            ]
+          }
+        ]
+      ],
+      plugins: [
+        ['@babel/plugin-transform-runtime', {
+          corejs: 3
+        }]
+      ]
+    };
```

Comment:
We want all these polyfils (core-js) and babel's runtime if we are deploying to the cdn.